src/main/resources/templates/pps/bankTransferPenaltyReference.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+  <title>
+    What type of penalty do you need to pay?
+  </title>
+</head>
+
+<div id="pps-penalty-reference"  layout:fragment="content">
+  <form id="ppspenaltyreference" th:action="@{''}" th:object="${penaltyReferences}" method="post" class="form">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <div th:replace="fragments/ppsGlobalErrors :: globalErrors"></div>
+
+        <h1 id="page-title " class="govuk-heading-xl">
+          <span id="page-title-heading">Find out which bank details to use</span>
+        </h1>
+        <p class="govuk-body">We need to ask you about your penalty reference, so that we can show you the correct bank details for the type of penalty you’re paying.</p>
+        <h2 class="govuk-heading-m">What does the penalty reference start with?</h2>
+        <p class="govuk-hint govuk-!-margin-top-3">For example, the penalty reference A1234567 starts with A.</p>
+        <div class="govuk-form-group" th:classappend="${#fields.hasErrors('selectedPenaltyReference')} ? 'govuk-form-group--error' : ''">
+          <fieldset class="govuk-fieldset">
+            <span class="govuk-error-message"
+                  id="selectedPaymentMethod-errorId"
+                  th:if="${#fields.hasErrors('selectedPenaltyReference')}"
+                  th:each="e : ${#fields.errors('selectedPenaltyReference')}" th:text="${e}">
+            </span>
+
+            <div class="govuk-radios" th:each="penaltyReference : ${availablePenaltyReference}" style="margin-bottom:1em;">
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input piwik-event"
+                       th:id="${penaltyReference}"
+                       name="available-payment-method"
+                       type="radio"
+                       th:value="${penaltyReference}"
+                       th:field="*{selectedPenaltyReference}"
+                       th:errorclass="govuk-error-message"
+                       th:attr="data-event-id=${penaltyReference}"/>
+                <label class="govuk-label govuk-radios__label" th:for="${penaltyReference}" th:text="${penaltyReference}">keyvalue</label>
+              </div>
+            </div>
+
+          </fieldset>
+        </div>
+        <div class="govuk-form-group">
+          <details class="govuk-details piwik-event" data-event-id="Details - Help with the penalty reference" data-module="govuk-details" aria-labelledby="penalty-reference">
```

Comment:
The `data-event-id="Details -` bit probably needs to be something like `data-event-id="Bank Transfer -`?